doc/man/common-footer.txt

```diff
@@ -0,0 +1,27 @@
+BUGS
+----
+If you encounter any issue or usability problem, please report it on the
+https://bugs.lttng.org/projects/lttng-ust[LTTng bug tracker].
+
+
+RESOURCES
+---------
+* http://lttng.org[LTTng project website]
+* http://lttng.org/docs[LTTng documentation]
+* http://git.lttng.org[Git repositories]
+* http://github.com/lttng[GitHub organization]
+* http://ci.lttng.org/[Continuous integration]
+* http://lists.lttng.org[Mailing list] for support and
+  development: `lttng-dev@lists.lttng.org`
+* irc://irc.oftc.net/lttng[IRC channel]: `#lttng` on `irc.oftc.net`
+
+
+COPYRIGHTS
+----------
+This {object-type} is part of the LTTng-UST project.
+
+LTTng-UST is distributed under the
+http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html[GNU General
```

Comment:
See https://github.com/lttng/lttng-ust/pull/20/commits/bb28c928bbf3c847d95957e1b979287e46807096#diff-d0ccc1de38c2c8391129c47ae45ba482R1 which is now a common copyrights section for everything but `lttng-gen-tp`, which has its own copyrights section: https://github.com/lttng/lttng-ust/pull/20/commits/bb28c928bbf3c847d95957e1b979287e46807096#diff-92213d61b8e08b1c8eb748b760006253R114